From f92639c8d08e97cd6f3d9608b17e4c222dc2e01d Mon Sep 17 00:00:00 2001
From: Camelia Groza <camelia.groza@nxp.com>
Date: Mon, 12 Aug 2019 15:58:07 +0300
Subject: [PATCH 016/245] sdk_dpaa: ceetm: update Makefile to use absolute
 include paths

commit d5ccd9e32ef769a359f85b35f9a3102eeba13830 from
http://source.codeaurora.org/external/qoriq/qoriq-components/linux.git

Explicitly mention the root source tree directory when building. This
allows the build process to start from a different location.

Signed-off-by: Camelia Groza <camelia.groza@nxp.com>
Signed-off-by: Zhantao Tang <zhantao.tang@windriver.com>
---
 drivers/net/ethernet/freescale/sdk_dpaa/Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/net/ethernet/freescale/sdk_dpaa/Makefile b/drivers/net/ethernet/freescale/sdk_dpaa/Makefile
index b709c8a9c72f..9bcd36cf4faa 100644
--- a/drivers/net/ethernet/freescale/sdk_dpaa/Makefile
+++ b/drivers/net/ethernet/freescale/sdk_dpaa/Makefile
@@ -18,7 +18,7 @@ ifeq ($(CONFIG_FSL_DPAA_1588),y)
 fsl_dpa-objs += dpaa_1588.o
 endif
 ifeq ($(CONFIG_FSL_DPAA_CEETM),y)
-ccflags-y += -Idrivers/net/ethernet/freescale/sdk_fman/src/wrapper
+ccflags-y += -I$(srctree)/drivers/net/ethernet/freescale/sdk_fman/src/wrapper
 fsl_dpa-objs += dpaa_eth_ceetm.o
 endif
 
-- 
2.17.1

